packages/rancher-monitoring/generated-changes/patch/templates/alertmanager/alertmanager.yaml.patch

```diff
@@ -9,7 +9,12 @@
    version: {{ .Values.alertmanager.alertmanagerSpec.image.tag }}
    {{- if .Values.alertmanager.alertmanagerSpec.image.sha }}
    sha: {{ .Values.alertmanager.alertmanagerSpec.image.sha }}
-@@ -25,8 +25,8 @@
+@@ -22,11 +22,13 @@
+   externalUrl: "{{ tpl .Values.alertmanager.alertmanagerSpec.externalUrl . }}"
+ {{- else if and .Values.alertmanager.ingress.enabled .Values.alertmanager.ingress.hosts }}
+   externalUrl: "http://{{ tpl (index .Values.alertmanager.ingress.hosts 0) . }}{{ .Values.alertmanager.alertmanagerSpec.routePrefix }}"
++{{- else if not (or (kindIs "invalid" .Values.global.cattle.url) (kindIs "invalid" .Values.global.cattle.clusterId)) }}
```

Comment:
`kindIs "invalid" {field}` is a nice little hack that allows us to not have to check `.Values.global` and `.Values.global.cattle` before looking at this field. If `.Values.global` or `.Values.global.cattle` do not exist, these statements will return true.